### PR TITLE
Changelog v1.17.0-rc2

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
...
+  * Display full name (#20171) (#20246)
+* BUGFIXES
+  * Fix commit status icon when in subdirectory (#20285) (#20385)
+  * Initialize cron last (#20373) (#20384)
+  * Set target on create release with existing tag (#20381) (#20382)
+  * Update xorm.io/xorm to fix a interpreting db column sizes issue on 32bit systems (#20371) (#20372)
+  * Make sure `repo_dir` is an empty directory or doesn't exist before 'dump-repo' (#20205) (#20370)
+  * Prevent context deadline error propagation in GetCommitsInfo (#20346) (#20361)
+  * Correctly handle draft releases without a tag (#20314) (#20335)
+  * Add write check for creating Commit status (#20332) (#20333)
```
Comment (delvh) [inline CHANGELOG.md]: Isn't that `breaking` instead?
The fact that this PR was already mentioned on Discord as the cause of unexpected behavior indicates that.

Comment (zeripath) [reply CHANGELOG.md]: I think we should make this BREAKING or SECURITY.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.17.0-rc2](https://github.com/go-gitea/gitea/releases/tag/v1.17.0-rc2) - 2022-07-13
+
+* SECURITY
+  * Use git.HOME_PATH for Git HOME directory (#20114) (#20293)
```
Comment (zeripath) [inline CHANGELOG.md:10]:
```suggestion
  * Use git.HOME_PATH for Git HOME directory (#20114) (#20293)
  * Add write check for creating Commit Statuses (#20332) (#20333)
```

File: CHANGELOG.md (hunk 1 above)
Comment (zeripath) [reply CHANGELOG.md]:
```suggestion
```

**Review by delvh — APPROVED** [review]

**Review by zeripath — APPROVED** [review]
I'm gonna approve as I think we should just release the next rc soon.

@go-gitea/mergers  double check before merging that the changelog is up-to-date and that there isn't anything drastically missing.

**Pull request merged**